lib/neo4j/migrations.rb

```diff
@@ -11,7 +11,8 @@ module Migrations
     class << self
       def check_for_pending_migrations!
         runner = Neo4j::Migrations::Runner.new
-        fail ::Neo4j::PendingMigrationError if runner.pending_migrations?
+        pending = runner.pending_migrations
+        fail ::Neo4j::PendingMigrationError, pending if pending.any?
```

Comment:
@cheerfulstoic check here:
https://github.com/neo4jrb/neo4j/pull/1309/files#diff-3ced7dc77d960f7c24fdee3fe17efe5cR40
It's passed as a parameter to the error, and joined with "\n"